index.html

```diff
@@ -18,7 +18,9 @@ <h1>Pizza Party
                 <a href="ourstory.html ">Our Story</a>
                 <a href="Menu.html ">Menu</a>
                 <a href="Reservations.html ">Reservations</a>
-                <a href="contactus.html ">Connect with Us!</a>
+                <a href="contactus.php">Connect with Us!</a>
```

Comment:
do we need two contactus file? 
contactus.html
contactus.php

Reply:
it was removed thats why its in red. i updated it to be a PHP file because it included PHP.

Reply:
opps! that makes sense.